examples/telemetry/grafana.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Grafana Deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: grafana
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: grafana
+  template:
+    metadata:
+      labels:
+        app: grafana
+    spec:
+      containers:
+      - name: grafana
+        image: grafana/grafana:latest
+        ports:
+        - containerPort: 3000
+        volumeMounts:
+          - mountPath: /etc/grafana/provisioning/datasources/grafana.yaml
+            name: grafana-config
+            subPath: grafana.yaml
+      volumes:
+      - name: grafana-config
+        configMap:
+          name: grafana-config
+
+---
+
+# Grafana Service
```

Comment:
I think Kubernetes encourages you to put a Service before the corresponding Deployment, so I would move this to the top.

Reply:
Done.